.github/workflows/build.yml

```diff
@@ -220,13 +220,26 @@ jobs:
         shell: bash
         working-directory: /workspace/gitpod
         run: |
+          RESULT=0
+          set -x
           leeway build dev:all \
             --docker-build-options network=host \
             --cache remote \
             -Dversion=$VERSION \
             -DlocalAppVersion=$VERSION \
-            -DimageRepoBase=eu.gcr.io/gitpod-core-dev/dev
+            -DimageRepoBase=eu.gcr.io/gitpod-core-dev/dev \
+            --report large-report.html || RESULT=$?
+
+          set +x
 
+          # capture logs for the dev/image build, they're too big for GITHUB_STEP_SUMMARY
+          pup -p -f large-report.html '[id="dev/image:docker"] + details + details' > remove.txt
+
+          # remove logs for the dev/image build output from the report file
+          grep -vxFf  remove.txt large-report.html > report.html
```

Comment:
This captures and removes most, but, not all docker build logs for the dev image. This is intentional, as the GITHUB_STEP_SUMMARY has a limited size.

Refer to [this build](https://github.com/gitpod-io/gitpod/actions/runs/4525181416) for an example of the Summary, see how it is helpful now, showing you the version of the new dev image. We can also [find the newly built image in GCR](https://console.cloud.google.com/gcr/images/gitpod-core-dev/eu/dev/dev-environment?project=gitpod-core-dev), but this saves effort/time.

Note: ChatGPT helped me simplify this step by solving the parsing problem with grep. :upside_down_face:

Reply:
And, [here is an example build where the dev image did not change](https://github.com/gitpod-io/gitpod/actions/runs/4525265289?pr=17019).